README.md

```diff
@@ -32,9 +32,9 @@ Depending on your distribution, you will need to run one of the following comman
 
 ## Note for Windows users running Linux on WSL
 
-ZCLI's credential manager has a dependency on Linux's windowing manager (X11) which will make some commands fail to run on WSL, unless you run a setup of WSL with support for X11/GUI apps. These commands include: `zcli login`, `zcli logout` and `zcli profiles`.
+ZCLI's credential manager has a dependency on Linux's windowing manager (X11) which will cause some commands fail to run on WSL, unless you run a setup of WSL with support for X11/GUI apps. These commands include: `zcli login`, `zcli logout` and `zcli profiles`.
 
-Features of ZCLI not dependent on ZCLI's credential manager should work in WSL2 w/o X11 support.
+Features of ZCLI not dependent on ZCLI's credential manager will work in WSL2 without X11 support.
 
 There is a number of workarounds:
```

Comment:
Not touched by this change, but if we're touching up the copy here, maybe modify this to `There are a number of workarounds:`